tests/webview/ui/test_content.py

```diff
@@ -452,22 +455,22 @@ def test_get_this_book(webview_base_url, selenium):
     if offline_zip_displayed:
         assert downloads.is_offline_zip_available
 
+# !!! old bug verification from March 2017. Test case was selected to be deleted. We should remove the whole function
```

Comment:
If it's old then remove it. We want to avoid keeping commented code as much as possible.

Reply:
Removed the old commented out function and all the comments regarding changes.